src/main/java/net/minecraftforge/fluids/FluidAttributes.java

```diff
@@ -446,13 +609,14 @@ protected Water(Builder builder, Fluid fluid)
         }
 
         @Override
-        public int getColor(BlockAndTintGetter world, BlockPos pos)
+        public int getColor(BlockAndTintGetter level, BlockPos pos)
         {
-            return BiomeColors.getAverageWaterColor(world, pos) | 0xFF000000;
+            return BiomeColors.getAverageWaterColor(level, pos) | 0xFF000000;
         }
 
         public static Builder builder(ResourceLocation stillTexture, ResourceLocation flowingTexture) {
-            return new Builder(stillTexture, flowingTexture, Water::new);
+            return new Builder(stillTexture, flowingTexture, Water::new).canExtinguish((state, entity) -> true);
```

Comment:
Is there a reason this canExtinguish -> true  is not in ForgeHooks#createVanillaFluidAttributes where all the other water specific properties are set?